prototyping/testing/searchTest.py

```diff
@@ -0,0 +1,63 @@
+import os
```

Comment:
I'd recommend using the [unittest](https://docs.python.org/2/library/unittest.html) framework for this (or select a different python unit test framework).

Reply:
@jaimehisao and I chose pytest because of it's ease of use with github actions. We did look into the unittest framework but we didn't see why we should choose that one. Could you please explain the shortcomings that pytest has?